cirq/ops/pauli_string.py

```diff
@@ -56,13 +58,26 @@ def from_single(qubit: raw_types.Qid,
     def coefficient(self) -> complex:
         return self._coefficient
 
+    def unit(self) -> UnitPauliStringT:
```

Comment:
This should just return a `PauliString` with coefficient equal to 1, similar to how `abs(int)` returns an `int`.

Reply:
I need this for `PauliSum.from_pauli_strings` and it's used in `_value_equality_values`. Maybe we can give it a different name?

Reply:
Given the current result type, this shouldn't be a public method. Users won't use it; it's too specific. It's an internal implementation detail.

If you need it in `from_pauli_strings`, define it in the pauli sum file instead of here. `_value_equality_values_` can be restored to its old form.

Reply:
done